File: src/main/kotlin/br/com/arthurfnsc/bandsapi/ApiApplication.java
package br.com.arthurfnsc.bandsapi;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class ApiApplication {

    //@Value("${spring.security.oauth2.client.registration.google.client-id}")
    //private String clientId;

    //@Value("${spring.security.oauth2.client.registration.google.client-secret}")
    //private String clientSecret;

    public static void main(String[] args) {
        //System.out.println("=> " + new ApiApplication().clientId);
        //System.out.println("==> " + new ApiApplication().clientSecret);

        SpringApplication.run(ApiApplication.class, args);
    }
}

File: src/main/kotlin/br/com/arthurfnsc/bandsapi/ApiApplication.kt
package br.com.arthurfnsc.bandsapi

import br.com.arthurfnsc.bandsapi.configs.AppProperties
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication
import springfox.documentation.swagger2.annotations.EnableSwagger2

@EnableConfigurationProperties(AppProperties::class)
@EnableSwagger2
@SpringBootApplication
class ApiApplication

fun main(args: Array<String>) {
    runApplication<ApiApplication>(*args)
}